hw/ip/otbn/dv/smoke/smoke_test.s

```diff
@@ -130,79 +130,79 @@ csrrw x0, 0x7d8, x0
 
 # sim environment provides a fixed value for RND (in other environment RND isn't
 # fixed so this test will have a different final state)
-# w2 = rnd = 0x99999999_99999999_99999999_99999999_99999999_99999999_99999999_99999999
+# w2 = rnd = 0xAAAAAAAA_99999999_AAAAAAAA_99999999_AAAAAAAA_99999999_AAAAAAAA_99999999
```

Comment:
Funny, this doesn't match smoke_expected.txt, and could easily get out of sync.  It may be better to just refer to that file for the expected values and avoid the duplication: saves time and confusion.

Reply:
I think it doesn't match smoke expected because in `call_stack_3`, `w2` gets combined with other values that are written back into `w2`.  Maybe it's worth adding a comment here saying that this is not the final value but instead it has an impact on the final value?

Reply:
This was because in the `standalone.py` -in which I ran the `smoke.elf` to get the register values per instruction- we were initialising a part of the sideload key with `0xbadf00d` instead of `0xbaadfood`. I agree to refer to the expected smoke test result file, however I believe this one is still nice to have since it shows the values each step of the way.